NINA-W1-OTP/components/ubx_phy_cal/ubx_phy_cal.c

```diff
@@ -154,6 +145,25 @@ const esp_phy_init_data_t* esp_phy_get_init_data()
     init_data[79] = 0x44; //channel 9, 10
     init_data[80] = 0x44; //channel 11
 
+    #if ESP_IDF_VERSION_MAJOR == 4 && ESP_IDF_VERSION_MINOR == 0
+    init_data[44] = 56; //target power 0
+    init_data[45] = 54; //target power 1
+    init_data[46] = 48; //target power 2
+    init_data[47] = 46; //target power 3
+    init_data[48] = 42; //target power 4
+    init_data[49] = 36; //target power 5
+
+    ##elif ESP_IDF_VERSION_MAJOR == 4 && ESP_IDF_VERSION_MINOR == 3
```

Comment:
Still you have two ## here, should only be one?

Reply:
[f160ba7](https://github.com/u-blox/u-blox-sho-OpenCPU/pull/7/commits/f160ba7da989ed5aecd0df2c719202732f8c3cec)